libs/dtm/dtm_recovery.py

```diff
@@ -271,3 +274,28 @@ def get_process_ids(health_obj, process):
         svc = switcher[process]
         fids = fids[svc]
         return True, fids
+
+    def perform_copy_objects(self, workload, que):
+        """
+        function to perform copy object for dtm test case in background
+        :param workload: Python dict containing source and destination bucket and object
+        :param que: Multiprocessing Queue to be used for returning values (Boolean,dict)
+        """
+        results = list()
+        failed_obj_name = ""
```

Comment:
This should be a list right?

Reply:
as we are breaking loop after first failure seen in operation, I am changing logic for results : list ==> Bool